cpp/src/gandiva/regex_functions_holder_test.cc

```diff
@@ -635,4 +635,134 @@ TEST_F(TestExtractHolder, TestErrorWhileBuildingHolder) {
   execution_context_.Reset();
 }
 
+class TestRegexpLikeHolder : public ::testing::Test {
+ protected:
+  ExecutionContext execution_context_;
+};
+
+TEST_F(TestRegexpLikeHolder, TestRegexpLikeUseNonParameter) {
+  EXPECT_OK_AND_ASSIGN(auto regex_like_holder, RegexpLikeHolder::Make("ast", false, ""));
+  auto& regex_like = *regex_like_holder;
```

Comment:
```suggestion
  EXPECT_OK_AND_ASSIGN(auto regexp_like_holder, RegexpLikeHolder::Make("ast", false, ""));
  auto& regexp_like = *regexp_like_holder;
```

Other tests have similar codes.